README.md

```diff
@@ -49,19 +49,27 @@ Additional Conda installation options available on the [gh-feedstock page](https
 | ------------------ | ---------------------------------------- |
 | `spack install gh` | `spack uninstall gh && spack install gh` |
 
+#### Webi
+
+| Install:                            | Upgrade:         |
+| ----------------------------------- | ---------------- |
+| `curl -sS https://webi.sh/gh \| sh` | `webi gh@stable` |
```

Comment:
Should this have a `\` before the pipe?

Reply:
Yes, the `\` prevents markdown recognizing the pipe as a column separator for the table.